src/kitovu/cli.py

```diff
@@ -37,8 +38,20 @@ def warn(self, message: str) -> None:
 
 
 @click.group()
-def cli() -> None:
-    pass
+@click.option('--loglevel',
+              type=click.Choice(['debug', 'info', 'warning', 'error', 'critical']),
+              default='info')
+def cli(loglevel: str) -> None:
+    level: int = getattr(logging, loglevel.upper())
+    if level == logging.DEBUG:
+        logformat = '%(asctime)s [%(levelname)5s] %(name)25s %(message)s'
+    else:
+        logformat = '%(message)s'
+
+    # PySMB has too verbose logging, we don't want to see that.
+    logging.getLogger('SMB.SMBConnection').propagate = False
```

Comment:
Wouldn't the SMB-Plugin be a better place for this? I'd expect only the SMB-Plugin to know anything about this